numpy/ma/core.py

```diff
@@ -6684,7 +6684,7 @@ def size(obj, axis=None):
 #####--------------------------------------------------------------------------
 #---- --- Extra functions ---
 #####--------------------------------------------------------------------------
-def where (condition, x=None, y=None):
+def where(condition, *args, **kwargs):
```

Comment:
You will probably need to rebase to use `_NoValue`. After updating your master branch, check out the working branch and do
```
git rebase master
git push -f origin HEAD
```
or equivalent.